README.md

```diff
@@ -6,76 +6,70 @@
 
 mc-components aims to provide primitive components to build user interfaces in MasterClass.
 
-## mc-components in your project
+## Getting Started
 This component library is most easily imported as a dependency.  You can use `yarn` or `npm` to bring it in to your project.  This ensures all required `js` and styles are included.
 
-`yarn add mc-components` or `npm install mc-components --save`
-
-Our components do **not** contain any styles.  The style library must also be imported manually into your root `scss` file.
-
-```scss
-@import 'mc-components';
-```
-
-### Font files
-We use Lato as our primary font for headings and body text in 300, 400, and 700 font weights.  You can import it in your css below, or import it into your asset pipeline.
-```
-@import url("https://fonts.googleapis.com/css?family=Lato:300,400,700");
-```
-
-### Standalone style usage
-If your project doesn't use yarn or npm for dependencies and you're only looking for masterclass css / style files, they can be imported manually by downloading the repository on github.  This will give you access to styles for buttons, the grid, typography, and more.  Save the contents of the `/dist/styles/scss/` folder to your own project, then import!
-
-**Note**: If your project doesn't have sass processing, there is a plain css file you can import here:
-
-```
-/dist/styles/css/mc-components.css
+```bash
+yarn add mc-components
```

Comment:
What if they don't know how to use `yarn`

Reply:
then you don't use mc-components